src/integrationTest/java/uk/gov/hmcts/divorce/document/content/NoticeOfProceedingContentIT.java

```diff
@@ -219,7 +219,7 @@ public void shouldSuccessfullyApplyContentFromCaseDataForGeneratingNoticeOfProce
         expectedEntries.put(IS_COURT_SERVICE, false);
         expectedEntries.put(IS_PERSONAL_SERVICE, false);
         expectedEntries.put(ACCESS_CODE, "ACCESS_CODE");
-        expectedEntries.put(URL_TO_LINK_CASE, "https://nfdiv-apply-for-divorce.aat.platform.hmcts.net/applicant2");
+        expectedEntries.put(URL_TO_LINK_CASE, "https://nfdiv-apply-for-divorce.aat.platform.hmcts.net/respondent");
```

Comment:
is this true for both sole and joint ? or for sole we need `/respondent` and for joint `/applicant2` ?

Reply:
joint NOP has its own content file called NoticeOfProceedingJointContent